Update cards-format.md (#2722)
MicrosoftDocs/msteams-docs: msteams-platform/task-modules-and-cards/cards/cards-format.md

}
```

### Full width Adaptive Cards
You can use this proprety to expand the width of an adaptive card and make use of addtional canvas space (see example below).

#### Constructing full width cards

> To make a full width Adaptive Card your app needs to include the following elements
> 
> * The `width` object inside of an `msteams` property in the card content should be set to `Full`

Reviewer: ```suggestion
To make a full width Adaptive card the `width` object in `msteams` property in the card content must be set to `Full`.
In addition, your app must include the following elements:
```
